deploy/crds/contrail.juniper.net_postgres_crd.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     listKind: PostgresList
     plural: postgres
     singular: postgres
-  scope: Namespaced
```

Comment:
Please update operator-sdk to version v0.17.1 as described in documentation.

Reply:
As per  document, for 2008R  we are using the older version, I have noticed u have changes done in master not in 2008R.

Reply:
but this is PR to master